packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 3.7.0 - Apr 06, 2022
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+New Features:
+- Provide multiple entry points of creating testing tenant/accounts to unblock user from M365 account issues, like M365 account does not have sideloading permission or user does not have M365 account at all. These entry points include an Add(+) button besides ACCOUNTS in sidebar, an new "create account" option in `Teams: Account` Command and improved Get Started page.
+
+Enhancement:
```

Comment:
Just to confirm, is this a new feature or improvement since we supported new features like Minimal framework after using Yeoman generator with latest SPFx?

Reply:
@xu-hong any thoughts here?